### PR TITLE
MUI Base `5.0.0-beta.18` [generated]

### PR DESCRIPTION
**Pull request by aerialist7**

### CODE REVIEW

File: kotlin-mui-base/src/jsMain/generated/mui/base/useSlider.types.kt [hunk 1]
```diff
@@ -156,24 +156,24 @@ external interface UseSliderReturnValue {
 
     /**
      * Resolver for the hidden input slot's props.
-     * @param otherHandlers props for the hidden input slot
+     * @param externalProps props for the hidden input slot
      * @returns props that should be spread on the hidden input slot
      */
-    var getHiddenInputProps: react.Props /* <TOther extends EventHandlers = {}>(otherHandlers?: TOther) => UseSliderHiddenInputProps<TOther> */
+    var getHiddenInputProps: react.Props /* <ExternalProps extends Record<string, unknown> = {}>(externalProps?: ExternalProps) => UseSliderHiddenInputProps<ExternalProps> */
```
Comment (turansky) [inline kotlin-mui-base/src/jsMain/generated/mui/base/useSlider.types.kt:162]: Strict handler expected in such cases

**Review by turansky — CHANGES_REQUESTED** [review]

File: kotlin-mui-base/src/jsMain/generated/mui/base/useSlider.types.kt (hunk 1 above)
Comment (aerialist7) [reply kotlin-mui-base/src/jsMain/generated/mui/base/useSlider.types.kt:162]: What do you mean? 
Here is only change in comment

Comment (turansky) [reply kotlin-mui-base/src/jsMain/generated/mui/base/useSlider.types.kt:162]: Strict type expected instead of comment

Comment (aerialist7) [reply kotlin-mui-base/src/jsMain/generated/mui/base/useSlider.types.kt:162]: Ok, will do it later in separate task. It is only version update with no changes in this place

**Pull request merged**